### PR TITLE
Adding in level and category fields in task object

### PR DESCRIPTION
**Pull request by Muralidhar22**
## Why updating this?
This PR is related to https://github.com/Real-Dev-Squad/todo-action-items/issues/72 for building out task level feature

Issue: https://github.com/Real-Dev-Squad/website-backend/issues/743

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
     <task_id>,
     <task_id>
   ],
+  "level": 1 | 2 | 3 | 4 | 5,
+  "category": "FRONTEND | BACKEND | SYSTEM DESIGN",
```
Comment (vinayak-trivedi) [inline tasks/README.md:28]: I think the category will not be just these three, there will be other categories as well

Comment (Muralidhar22) [reply tasks/README.md:28]: okay should I give any generic type as string in UPPERCASE, <architecture>

**Review by Maheima — APPROVED** [review]
LGTM

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

**Review by rupal-2505 — APPROVED** [review]

**Pull request merged**